packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -57,15 +59,18 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
 
       this.webSocketManager = new WebSocketManager({webex: this.$webex});
 
-      this.connectionService = new ConnectionService(
-        this.webSocketManager,
-        this.getConnectionConfig()
-      );
+      this.connectionService = new ConnectionService({
+        webSocketManager: this.webSocketManager,
+        onReRegister: this.register.bind(this),
```

Comment:
Passing reRegister as a callback to the connection-service, felt this would be the best way so that we do not have any circular dependencies. Open to any other suggestions however.